firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigContainer.java

```diff
@@ -143,6 +167,64 @@ public boolean equals(Object o) {
     return containerJson.toString().equals(that.toString());
   }
 
+  /**
+   * @param other The other {@link ConfigContainer} against which to compute the diff
+   * @return The set of config keys that have changed between the this config and {@code other}
+   * @throws JSONException
+   */
+  public Set<String> getChangedParams(ConfigContainer other) throws JSONException {
+    // Make a deep copy of the other config before modifying it
+    JSONObject otherConfig = ConfigContainer.deepCopyOf(other.containerJson).getConfigs();
+
+    Set<String> changed = new HashSet<>();
+    Iterator<String> keys = this.getConfigs().keys();
+    while (keys.hasNext()) {
```

Comment:
It would prob be easier to just call `other.remove(key)` after calling `changed.add(key)`

Reply:
I think for now we're just going to leave it how it is since it doesn't effect the end result and touch back upon it when we our fast-follows and re-factor parts of Realtime